website/src/components/blog.tsx

```diff
@@ -0,0 +1,58 @@
+export enum ChipColor {
+  blue = "blue-chip",
+  green = "green-chip",
+  yellow = "yellow-chip",
+  purple = "purple-chip",
+}
+
+type Props = {
```

Comment:
Favor interface over type aliases for static types.

Reply:
sure